tests/lib/ecma-features.js

```diff
@@ -75,17 +75,18 @@ describe("ecmaFeatures", function() {
         // Uncomment and fill in filename to focus on a single file
         // var filename = "jsx/invalid-matching-placeholder-in-closing-tag";
         var feature = path.dirname(filename),
+            permissiveValue = feature !== "impliedStrict",
```

Comment:
Ooh, I think you're doing this because the implied strict tests are failing. Please revert this change and fix the tests.